slither/utils/expression_manipulations.py

```diff
@@ -74,6 +74,11 @@ def apply_copy(
     def copy_expression(
         self, expression: Expression, true_expression: Expression, false_expression: Expression
     ) -> None:
+        # case of unused
+        # (, var) = func()
+        if not expression:
+            return
```

Comment:
Wouldn't that mean `Expression` can be None, which means type of `expression` should be `Optional[Expression]` ?